tasks/add_new_prebid_partner.py

```diff
@@ -180,10 +185,11 @@ def create_line_item_configs(prices, order_id, placement_ids, bidder_code,
     price_str = num_to_str(micro_amount_to_num(price))
 
     # Autogenerate the line item name.
-    line_item_name = u'{bidder_code}: HB ${price}'.format(
+    line_item_name = u'{bidder_code}: HB {price}'.format(
       bidder_code=bidder_code,
-      price=price_str
+      price=u'\u00a3' + price_str
```

Comment:
Please revert these changes. We can open a separate issue to allow customizing line item names or to use smarter naming based on the chosen currency. However, it's not generalizably useful to just replace hardcoded `$` with `£`.